packages/contracts/test/unit/BaseStrategy.t.sol

```diff
@@ -450,4 +451,105 @@ contract BaseStrategyTest is TestWithERC1820Registry {
 
         assertEq(baseStrategy.minimumBetweenExecutions(), minReportInterval);
     }
+
+    function testHealthCheckPass() public {
+        uint256 _120_USDT = 120 * 10 ** 18;
```

Comment:
Try to use fuzz testing where is possible
```suggestion
    function testHealthCheckPass(uint256 amount) public {
```

Reply:
Same for rest of the tests

Reply:
fixed.